HomeWork_20/HW_20.py

```diff
@@ -0,0 +1,9 @@
+import sqlite3
```

Comment:
This file can be removed since it does nothing but connect to a database which is not a part of the task